### DISCUSSION TITLE
[Superset SDK] getScrollSize() not working as expected

### DISCUSSION THREAD

**Discussion by vleevermind** [apache/superset#28101] [Q&A / Help]
Hi,
Im newbie to Superset.

Im trying to get the height of the dashboard to fix my embed iframe height.

Here is my code:
 ```
                let embed_dashboard = await embedDashboard({
                    id: embed_data.uuid,
                    supersetDomain: 
                    mountPoint: 
                    fetchGuestToken: () => 
                    dashboardUiConfig: {
                        hideTitle: true,
                        hideTab: true,
                    },
                });

                let nIntervId = setInterval(async () => {
                    let { width, height } = embed_dashboard.getScrollSize();
                    console.log(height);
                }, 1000);
```

I dont know this is a issue or Im using it wrong but it keeps logging _undefined_ in my embed webpage and the height is set to iframe default height 150px.

Please help me.
THanks alot

**Comment by Finesim97** [comment]
It is an async function, so you need to use `await` for `getScrollSize`. However, this only returns the current size (Dashboards contents are responsive).
